example/pubspec.yaml

```diff
@@ -0,0 +1,28 @@
+name: example
+description: A new Flutter project.
+publish_to: 'none'
+version: 0.1.0
```

Comment:
Version is not neeed in `example/pubspec.yaml`.

Reply:
Removed it